### DISCUSSION TITLE
Cannot redirect back to main "category" page

### DISCUSSION THREAD

**Discussion by tranlv** [mmistakes/minimal-mistakes#2552] [Q&A] [answered]
When  I click on the link "posts" , it did not redirect back to  main "category" page, but return to "404 not found" page

https://distributedsystemsblog.com/posts/zombie-process/
https://distributedsystemsblog.com/categories/#posts

<img width="1045" alt="Screenshot 2020-06-06 at 3 28 37 PM" src="https://user-images.githubusercontent.com/12371875/83938842-9ae41680-a80a-11ea-8b30-da9365759558.png">

**Comment by iBug** [comment, marked as answer]
If you're building on GitHub Pages then you should manually create the categories page (and the tags page) yourself:

<https://mmistakes.github.io/minimal-mistakes/docs/configuration/#archive-settings>

**Comment by tranlv** [comment]
@iBug Thanks, how do I hide the link , in this page https://mmistakes.github.io/minimal-mistakes/docs/installation/ ; he does not have the link on the top

**Comment by mmistakes** [comment]
Breadcrumb navigation is beta and unreliable. It needs a lot of configuration and additional requirements like having all of the necessary category, topic, and or archive pages built. 

I personally wouldn’t encourage using it as if your permalink structure doesn’t match a specific format you’re going to have 404 errors.

https://mmistakes.github.io/minimal-mistakes/docs/navigation/#breadcrumbs-beta

**Comment by tranlv** [comment]
@mmistakes thanks, closing!
